File: baejoon/1003(피보나치 함수).cpp
#include<iostream>
using namespace std;

typedef struct a {
	int zero_count=0;
	int one_count=0;
}a;

int main() {
	int tc = 0;
	a arr[41];
	cin >> tc;
	for (int test_case = 0; test_case < tc; ++test_case) {
		int num = 0;
		cin >> num;
		arr[0].zero_count = 1;
		arr[0].one_count = 0;
		arr[1].zero_count = 0;
		arr[1].one_count = 1;
		for (int i = 2; i <= num; ++i) {
			arr[i].zero_count = arr[i - 1].zero_count + arr[i - 2].zero_count;
			arr[i].one_count = arr[i - 1].one_count + arr[i - 2].one_count;
		}
		cout << arr[num].zero_count << " " << arr[num].one_count<<endl;
	}
	return 0;
}

File: baejoon/1003(피보나치 함수).py
tc=int(input())
onecount=[0]*41
zerocount=[0]*41
for i in range(0,tc):
    num=int(input())
    zerocount[0]=1
    onecount[0]=0
    zerocount[1]=0
    onecount[1]=1
    for i in range(2,num+1):
        zerocount[i]=zerocount[i-1]+zerocount[i-2]
        onecount[i]=onecount[i-1]+onecount[i-2]
    print(zerocount[num],onecount[num],sep=" ")
    /seq 는 print(a,b,seq=" ") a" "b로 
    
     
